composer.json

```diff
@@ -18,7 +18,8 @@
         "php": ">=5.5.9",
         "illuminate/support": "5.*",
         "nesbot/carbon": "^1.20",
-        "mailgun/mailgun-php": "^1.8"
+        "mailgun/mailgun-php": "^2.0",
+        "php-http/guzzle6-adapter": "^1.0"
```

Comment:
One should not put this requirement here. This should be added in the user's application. We should let the user decide what library to use to send http messages.